deepchem/metalearning/tests/test_maml_pytorch.py

```diff
@@ -0,0 +1,122 @@
+import deepchem as dc
+import numpy as np
+import pytest
+from flaky import flaky
+import unittest
+
+try:
+    import torch
+    import torch.nn.functional as F
+
+    has_pytorch = True
+except:
+    has_pytorch = False
+
+
+class TestMAML(unittest.TestCase):
+
+    @flaky
+    @pytest.mark.torch
+    def test_maml_pytorch(self):
+
+        class SineLearner(dc.metalearning.TorchMetaLearner):
```

Comment:
you can try `dc.metalearning.torch_maml.TorchMetaLearner` if the tests fail again